paragraph/session.py

```diff
@@ -71,14 +77,29 @@ def _count_usages(output: Iterable[Variable]) -> Dict[Variable, int]:
     return usage_counts
 
 
+def _get_arguments(var: Variable, cache: Dict[Variable, Any], usage_counts: Dict[Variable, int], output):
```

Comment:
Should we add a type annotation for the return value?

Reply:
Good catch, will do